doc/ap_verify_hits2015/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _ap_verify_hits2015-package:
+
+##################
+ap_verify_hits2015
+##################
+
+The ``ap_verify_hits2015`` package represents a small subset of the DECam `HiTS <http://astroinf.cmm.uchile.cl/the-high-cadence-transient-survey/>`_ survey.
+It is a good example dataset for difference imaging with DECam.
+
+Project info
+============
+
+Repository
+   https://github.com/lsst/ap_verify_hits2015
+
+.. Datasets do not have their own (or a collective) Jira components; by convention we include them in ap_verify
+
+Jira component
+   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM %20AND%20 component %3D ap_verify %20AND%20 text ~ "hits2015">`_
+
+Dataset Contents
+================
+
+This package provides complete data for three fields from the 2015 HiTS survey.
+It contains:
+
+* raw images from the ``Blind15A_26``, ``Blind15A_40``, and ``Blind15A_42`` survey fields, in g band.
+* biases (``zci``), flats (``fci``), and corresponding weight maps (``zcw``, ``fcw``) in all DECam bands (ugriz, Y, VR). [Note: weights are not yet supported by the Stack]
+* illumination corrections (``ici``) in g, r, and i band. [Note: illumcors are not yet supported by the Stack]
+* defect masks (``bpm``) from December 5, 2014.
+* reference catalogs for Gaia DR1 and Pan-STARRS1, covering the raw images' footprint.
+* templates coadded from HiTS 2014 data, covering the raw images' footprint.
+
+Intended Use
+============
+
+This dataset is designed for medium-scale difference imaging analysis, using "deep" coadd templates, by ``ap_verify``.
+It can also be used for difference imaging with single-image templates, but the single-season coverage of the raw data means the resulting difference images will be low quality.
```

Comment:
I almost hate to point out that, at present, the resulting difference images aren't exactly high quality with the deep coadd templates either... (perhaps reword to say "may be low quality")